python/cudf/tests/test_string.py

```diff
@@ -751,19 +750,13 @@ def test_string_groupby_key(str_data, str_data_raise, num_keys):
     pdf['a'] = other_data
     gdf['a'] = other_data
 
-    expectation = raise_builder(
-        [str_data_raise],
-        GDFError
-    )
```

Comment:
If you're removing this you should update the parameters above to remove the raise parameter from it.